Assets/Tests/EditMode/AuthTests/JwtStorage/JwtTokenTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using NUnit.Framework;
+using Src.AuthController.JwtManagement.Tokens;
+
+namespace Tests.EditMode.AuthTests.JwtStorage
+{
+    public class JwtTokenTests
+    {
+        const string TokenStub = "abc";
+        const string TokenReplacementStub = "cba";
+        
+        [Test]
+        public void Valid_ShouldRecognizeContainingData([Values(360, 5000, 28000)] int issued, [Values(600, 150, 5002)] int expires)
```

Comment:
PP: I think name of the method should emphasize that Valid compares issued and expired time. Possible name is "Valid_ShouldTellWheatherIssuedTimeIsLessThanExpiresTime" or something more readable.